libs/platform/user/ebpf_platform_user.cpp

```diff
@@ -226,13 +226,20 @@ ebpf_random_uint32()
 }
 
 uint64_t
-ebpf_query_interrupt_time_precise()
+ebpf_query_time_since_boot(bool include_suspended_time)
 {
     uint64_t interrupt_time;
-    // QueryInterruptTimePrecise returns A pointer to a ULONGLONG in which to receive the interrupt-time count in system
-    // time units of 100 nanoseconds.
-    // https://docs.microsoft.com/en-us/windows/win32/api/realtimeapiset/nf-realtimeapiset-queryinterrupttimeprecise.
-    QueryInterruptTimePrecise(&interrupt_time);
+    if (include_suspended_time) {
+        // QueryUnbiasedInterruptTimePrecise returns A pointer to a ULONGLONG in which to receive the interrupt-time
+        // count in system time units of 100 nanoseconds.
```

Comment:
Add that `The unbiased interrupt-time count does not include time the system spends in sleep or hibernation.`

Reply:
Same for the KM please.